_commands/scripting/yes.md

```diff
@@ -4,21 +4,52 @@
 yes
 ---
 
-`yes` will continuosly output lines with `y`. Can be useful for automating
+`yes` will continuosly output lines with `y`until user uses ctrl+c to stop. `yes` Can be useful for automating
 scripts.
 
 ~~~ bash
 $ yes
 y
 y
+^C
+$ yes n
+n
+n
+n
+^C
 ~~~
 
-<!--more-->
-
```

Comment:
You also need to include the `<!--more-->` somewhere around this point, otherwise you get an entry that looks like this ...
![screen shot 2016-03-25 at 6 28 57 pm](https://cloud.githubusercontent.com/assets/7095187/14055328/77e80e78-f2b7-11e5-98c7-4dc692207bee.png)